src/flows/shared/steps/OrganizeCollection/Sidebar/Sidebar.tsx

```diff
@@ -44,10 +48,22 @@ function Sidebar() {
     setNftsIsSelected(sidebarNfts, false);
   }, [sidebarNfts, setNftsIsSelected, unstageNfts]);
 
+  const handleRefreshWalletClick = useCallback(async () => {
+    await refreshOpenseaSync({ skipCache: true });
+    void refreshUnassignedNfts({ skipCache: false });
```

Comment:
any reason we don't need to refresh unassigned NFTs with `skipCache: true`? is it bc the opensea step resets the cache?